lemur/certificates/schemas.py

```diff
@@ -353,6 +353,12 @@ def handle_subject_details(self, data):
                 data.pop("organization", None)
                 data.pop("organizational_unit", None)
 
+        # Removing optional fields if None, else it complains in de-serialization
+        if "location" in data and data["location"] is None:
```

Comment:
should this be expanded to all `key: None` in `data`

Reply:
The other fields are mandatory, but it could be safe to remove None and rely on Input schema to handle as needed.